.gitignore

```diff
@@ -21,6 +21,7 @@ build/
 bld/
 [Bb]in/
 [Oo]bj/
+LocalPackages/
```

Comment:
Yes this is intentional since LocalPackages contains the locally generated nupkg files that should be ignored by git

---
In reply to: [70352251](https://github.com/Microsoft/xunit-performance/pull/159#discussion_r70352251) [](ancestors = 70352251)